addons/website_sale/static/src/scss/product_tile.scss

```diff
@@ -1523,6 +1598,34 @@ $_wsale_products_roundness_map: (
 .o_wsale_products_opt_thumb_2_3 {
     --o-wsale-card-thumb-aspect-ratio: 2/3;
 }
+.o_wsale_products_opt_thumb_1_1 {
+    --o-wsale-card-thumb-aspect-ratio: 1/1;
+}
+
```

Comment:
Not sure why we need to add this, `.o_wsale_products_opt_thumb_1_1` doesn't seem to be applied and it's equal do default value anyway